ContactDetailsApi.Tests/MockWebApplicationFactory.cs

```diff
@@ -65,26 +32,23 @@ public class MockWebApplicationFactory<TStartup>
 
         };
 
-        public HttpClient Client { get; private set; }
-        public IDynamoDbFixture DynamoDbFixture { get; private set; }
-        public ISnsFixture SnsFixture { get; private set; }
+        public HttpClient Client { get; protected set; }
+        public IDynamoDbFixture DynamoDbFixture { get; protected set; }
+        public ISnsFixture SnsFixture { get; protected set; }
 
-        private static List<ApiVersionDescription> _apiVersions { get; set; }
-        private const string ApiName = "Contact Details API";
+        protected static List<ApiVersionDescription> _apiVersions { get; set; }
```

Comment:
What is the intention behind changing these `privates` to `protected`?

Reply:
Some of them were needed in the derived class (MockWebApplicationWithMiddleware), but not all of them so I've reverted the changes that don't have to be used